src/libraries/Native/AnyOS/System.Globalization.Native/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+project(System.Globalization.Native C)
```

Comment:
I don't know why this file wasn't marked as renamed as I used `git mv` to move all the files. But this file contains a few edits from the original one.

Reply:
Same thing I noticed previously and found that this is because of default [diff rename limit of 400](https://github.com/git/git/blob/b06d3643105c8758ed019125a4399cb7efdcce2c/diff.c#L31), which can be changed with `git config diff.renamelimit` locally (e.g. set to -1 for highest possible value). However, that value will not be respected on server side (e.g. GitHub's diff viewer uses its own value and may display the rename differently than the local git-diff's output).

afaik, one good convention for better tracking is to have separate commits for renaming the file, and making changes to it.

Reply:
Yeah. Locally it showed as renamed so that was weird. I was going to do that but at the end it shows the diff the same. You would have to review per commit.

Reply:
True, it is just good for when we look at the history of individual file, it links it with previous location and enhances tracking.